shotover-proxy/src/transforms/mod.rs

```diff
@@ -362,6 +363,10 @@ impl<'a> Wrapper<'a> {
 
         let transform_name = transform.get_name();
         let chain_name = self.chain_name.clone();
+        debug!(
```

Comment:
Leftovers?

Reply:
I don't think this is necessary. We know what order the transform chain is being performed in from the topology file.